# Exception Hadling in py..
#try ... except..

# def findReciprocal(value):
#     try:
#         print("Value:",value)
#         r=1/value
#         print("The reciprocal of",value,"is",r,"/n")
#     except:
#         print()
#
# findReciprocal('hello')


